.github/CODEOWNERS

```diff
@@ -7,9 +7,12 @@
 *    @ozym @GeoNet/dmc
 
 # Codes and tests for downstream applications
+go.mod    @GeoNet/platform-team @GeoNet/ssd-team-developer
+go.sum    @GeoNet/platform-team @GeoNet/ssd-team-developer
 /cmd/    @GeoNet/platform-team @GeoNet/ssd-team-developer
 /internal/    @GeoNet/platform-team @GeoNet/ssd-team-developer
 /meta/    @GeoNet/platform-team @GeoNet/ssd-team-developer
+/resp/    @GeoNet/platform-team @GeoNet/ssd-team-developer
```

Comment:
happy with everything else, but the `/resp` folder also contains sensor metadata (in auto, files and nrl), so I think the ownership for this shall be extended to also include SOD team?

Reply:
Yup, good catch. Updated.